pandas/core/arrays/base.py

```diff
@@ -1994,7 +1994,7 @@ def _hash_pandas_object(
         ...                                      hash_key="1000000000000000",
         ...                                      categorize=False
         ...                                      )
-        array([11381023671546835630,  4641644667904626417], dtype=uint64)
+        array([ 6238072747940578789, 15839785061582574730], dtype=uint64)
```

Comment:
Would be good to note that hashing values for masked EA types changed in `Other API Changes`

Reply:
added